### PR TITLE
Fixes #226

### LINKED ISSUE #226: CClaimTrieCache::removeClaim does not need the height parameter
... since the claim object itself knows the correct height. Please use the one on the claim object and remove the parameter and simplify the code accordingly.

### PR DESCRIPTION
**Pull request by arrodriguez**

### CODE REVIEW

**Comment by BrannonKing** [comment]
I apologize that we have a silly formatting requirement at the moment. Can you run this command and submit one more time?
```
git diff -U0 master -- '*.h' '*.cpp' | ./contrib/devtools/clang-format-diff.py -p1 -i
```

**Comment by arrodriguez** [comment]
Hi Thank you for your review.
Before sending the pull request i execute all travis stages (for linux) .
The problem is  the version of clang-format version in my OS is 3.8 and the
trusty version is 3.4. I think that command output is different between
version. I really dont know why. I will try again in a container.

**Comment by BrannonKing** [comment]
I want to merge this today. Any luck with the formatting? Can you rebase on current master?

**Comment by arrodriguez** [comment]
Hi @BrannonKing i will try to make the today. 

Thanx for the patience

**Comment by arrodriguez** [comment]
Hi @BrannonKing ,  i think that all is covered here now. I did also rebase master branch with this PR.

Any feedback is welcome.

Regards,

File: src/claimtrie.cpp [hunk 1]
```diff
@@ -1708,7 +1708,12 @@ void CClaimTrieCache::removeFromExpirationQueue(const std::string& name, const C
         if (itQueue != itQueueRow->second.end())
         {
             itQueueRow->second.erase(itQueue);
+            std::cerr << __func__ << ": Successfully remove" << name << " from Expiration Cache ." << std::endl;
+        } else {
+            std::cerr << __func__ << ": Cannot remove " << name << " from Expiration Cache ." << std::endl;
         }
+    } else {
```
[the remark this answers is not in the document]
Comment (arrodriguez) [reply src/claimtrie.cpp]: It makes sense, i think i misunderstood your request in the first comment.

File: src/claimtrie.cpp [hunk 2]
```diff
@@ -1708,7 +1708,12 @@ void CClaimTrieCache::removeFromExpirationQueue(const std::string& name, const C
         if (itQueue != itQueueRow->second.end())
         {
             itQueueRow->second.erase(itQueue);
+            std::cerr << __func__ << ": Successfully remove" << name << " from Expiration Cache ." << std::endl;
+        } else {
+            std::cerr << __func__ << ": Cannot remove " << name << " from Expiration Cache ." << std::endl;
```
[the remark this answers is not in the document]
Comment (arrodriguez) [reply src/claimtrie.cpp]: Ok

File: src/claimtrie.cpp [hunk 3]
```diff
@@ -1708,7 +1708,12 @@ void CClaimTrieCache::removeFromExpirationQueue(const std::string& name, const C
         if (itQueue != itQueueRow->second.end())
         {
             itQueueRow->second.erase(itQueue);
+            std::cerr << __func__ << ": Successfully remove" << name << " from Expiration Cache ." << std::endl;
```
[the remark this answers is not in the document]
Comment (arrodriguez) [reply src/claimtrie.cpp]: Ok.

**Comment by tzarebczan** [comment]
hey @arrodriguez, thanks so much for this PR and congrats on getting it merged into our codebase! If you haven't already, check out [our contributing guide](https://lbry.tech/contribute). 

Also, can we offer you some [LBC in appreciation](https://lbry.io/faq/appreciation) for your contribution?

**Comment by arrodriguez** [comment]
Yes , please i would like to receive some LBC for the contribution.

Here is my address:

bRu6UAubBgfNUTrPyeYyeGLNE6q4kHVFUP

i will try to do more contributions whenever i can.

Thanx a lot!

**Comment by arrodriguez** [comment]
Yes , please i would like to receive some LBC for the contribution.

Here is my address:

bRu6UAubBgfNUTrPyeYyeGLNE6q4kHVFUP

i will try to do more contributions whenever i can. 

Thanx a lot!

**Pull request merged**